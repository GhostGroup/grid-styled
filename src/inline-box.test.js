import React from "react";
import { shallow, mount} from "enzyme";
import InlineBox from "./inline-box";

describe("grid", () => {
  it("renders", () => {
    const tree = shallow(<InlineBox />);
    expect(tree).toMatchSnapshot();
  });

  it("has a classname", () => {
    const div = shallow(<InlineBox />);
    expect(div.props().className).toBeDefined();
  });

  it('can use a different component with the "is" prop', () => {
    const tree = mount(
      <InlineBox m={2} px={3} is="section">
        Test
      </InlineBox>
    );
    expect(tree).toMatchSnapshot();
  });
});