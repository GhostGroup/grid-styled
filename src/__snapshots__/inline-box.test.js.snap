// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`grid can use a different component with the "is" prop 1`] = `
.c0 {
  display: inline-block;
  vertical-align: top;
}

.c1 {
  box-sizing: border-box;
  margin: 8px;
  padding-left: 16px;
  padding-right: 16px;
}

<InlineBox
  is="section"
  m={2}
  px={3}
>
  <Box
    className="c0"
    is="section"
    m={2}
    px={3}
  >
    <styled.section
      className="c0"
      m={2}
      px={3}
    >
      <section
        className="c0 c1"
      >
        Test
      </section>
    </styled.section>
  </Box>
</InlineBox>
`;

exports[`grid renders 1`] = `
.c0 {
  display: inline-block;
  vertical-align: top;
}

<Box
  className="c0"
  is="div"
/>
`;
